src/module/sheet-skill-actions.ts

```diff
@@ -37,7 +56,18 @@ Hooks.on('renderActorSheet', async (app: ActorSheet, html: JQuery<HTMLElement>)
   const skillActions: Array<SkillAction> = initializeSkillActions(app.actor).sort((a, b) => {
     return a.label > b.label ? 1 : -1;
   });
-  const skillActionHtml = await renderTemplate(tpl, { skills: skillActions });
+  const skillActionHtml = $(await renderTemplate(tpl, { skills: skillActions }));
+
+  skillActionHtml.on('click', '.item-image', (e) => {
+    const itemLabel = e.currentTarget.dataset.itemLabel;
+    if (!itemLabel) return;
+
+    const actionItem = ActionsIndex.instance.get(itemLabel);
+    if (!actionItem) return;
+
+    const ownedItem = new (actionItem.constructor as ItemConstructor)(actionItem.toJSON(), { parent: app.actor });
+    ownedItem.toChat();
```

Comment:
This would look nicer if each `SkillAction` would render one by one. Honestly I'm not quite sure if this doesn't leak memory or what are the best practices in foundry to attach events?